test/reduce.js

```diff
@@ -71,7 +72,7 @@ describe('reduce', function() {
   });
 
   it('short circuits with reduced', function() {
-    var addWithMaxOf10 = function(acc, val) {return acc + val > 10 ? R.reduced(acc) : acc + val;};
+    var addWithMaxOf10 = _xwrap(function(acc, val) {return acc + val > 10 ? R.reduced(acc) : acc + val;});
```

Comment:
Is there a way of setting this test up without using internal functions?

Reply:
No, unless we export or duplicate it.

Reply:
There should be a way to test for everything using our public API. Otherwise we are in a situation where users can't properly use a function without knowing the internals of our system - in this case _xwrap. 

If a change is solely internal it shouldn't affect the existing tests at all. If it changes the API, we should be able to capture that change in a test that demonstrates the correct behavior as it pertains to the public API change.